mediator/Makefile

```diff
@@ -9,7 +9,7 @@ ifndef PIO_INCLUDE_DIR
 $(error PIO_INCLUDE_DIR not set)
 endif
 
-ifdef INTERNAL_PIO_INIT
+ifeq ($(INTERNAL_PIO_INIT),1)
```

Comment:
I think that 0/1 indicates FALSE and TRUE. It is not related with the PIO version hopefully in the near future. Since this part of the code will be unnecessary when HAFS also adopts the external PIO installation, I think that we could keep it like as it is.